generators/form/templates/formComplex.js.ejs

```diff
@@ -1,4 +1,73 @@
-<%- include('partials/formHeader', {formNumber: formNumber}); %>
+import _ from 'lodash/fp';
+import pensionsSchema from 'vets-json-schema/dist/21P-527EZ-schema.json';
```

Comment:
Understood on the first point. I'll add back a commented-out import of the app-specific schema.

If I understand the second point, it certainly sounds like the `definitions.json` should be the maintained as the de-facto standard definitions for things like name fields, contact fields, and anything else that is used across different forms. But I have no idea what's currently in there and if it can be used in lieu of the two schemas I'm currently using.

Reply:
Thanks Erik, it looks like all of the definitions can be imported from the [common definitions file](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js), except for email, which we can define in line as:
```
type: 'string',
format: 'email'
```

Reply:
Cool! I'll give that a shot!